README.md

```diff
@@ -13,7 +13,7 @@ to `puppetserver` to switch to the JVM-based Puppet Server.
 
 Many puppet.conf options for agents, masters and other are parameterized, with
 class documentation provided at the top of the manifests.
-`server_additional_settings` can be used to supply master options that are
+The `*additional_settings` parameters can be used to supply options that are
 missing parameters.
```

Comment:
Ok I was just trying not to make a list ;) Explicit is best I think, let me try.